File: FCB-LaMasia.Web/wwwroot/Scripts/Models/Player.js
class Player extends User
{
    constructor(json)
    {
        if (json)
        {
            this.Position = json.position;
            this.Height = json.height;
            this.Weight = json.weight;
            this.Team = json.team;
        }
        else
        {
            this.Position = "";
            this.Height = 0;
            this.Weight = 0;
            this.Team = "";
        }
    }
}

File: FCB-LaMasia.Lib/Models/Player.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace FCB_LaMasia.Lib
{
    public class Player : User
    {
        public enum Position
        { Goalkeeper,
            Defender,
            Midfielder,
            Forward }
        public float Height { get; set; }
        public float Weight { get; set; }
        public Guid TeamId { get; set; }
        [JsonIgnore]
        public Team Team { get; set; }
    }
}
